src/utils.ts

```diff
@@ -161,6 +161,20 @@ export function getPublicFunctionLineNumber(
   return 0;
 }
 
+export function encodeInputArgsForViewRequest(type: string, value: string) {
+  if (type.includes("vector")) {
+    // when it's a vector, we support both hex and javascript array format
+    return value.trim().startsWith("0x")
+      ? value.trim()
+      : encodeVectorForViewRequest(type, value);
```

Comment:
this might not work well, cuz imagine if i input random string like "aptos" to the view function thats taking vector<u8>, it will end up in `encodeVectorForViewRequest`, which doesn't work in this case. 

<img width="1118" alt="Screenshot 2023-06-05 at 3 03 26 PM" src="https://github.com/aptos-labs/explorer/assets/121921928/7ca00b56-15b6-4278-8ce9-8bfc437cf8c5">

Reply:
I pushed a new commit to add some data validaiton.